### PR TITLE
#169 :- Quick fix for Resolver inspection adds implement directive to the next line

### PR DESCRIPTION
**Pull request by konarshankar07**
### Description (*)

This PR will add following interface to implement resolver
1. `Magento\Framework\GraphQl\Query\ResolverInterface`
2. `Magento\Framework\GraphQl\Query\Resolver\BatchResolverInterface`
3. `Magento\Framework\GraphQl\Query\Resolver\BatchServiceContractResolverInterface`

### Fixed Issues (if relevant)

1. magento/magento2-phpstorm-plugin#169: Quick fix for Resolver inspection adds implement directive to the next line

### Screenshot
image

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### Note
I'm not able to reproduce the issue with the interface applied to the next line. I will check again this issue

### CODE REVIEW

File: src/com/magento/idea/magento2plugin/magento/files/GraphQlResolver.java [hunk 1]
```diff
@@ -6,5 +6,7 @@
 
 public class GraphQlResolver {
     public static final String RESOLVER_INTERFACE = "\\Magento\\Framework\\GraphQl\\Query\\ResolverInterface";
+    public static final String BATCH_RESOLVER_INTERFACE = "\\Magento\\Framework\\GraphQl\\Query\\Resolver\\BatchResolverInterface";
+    public static final String BATCH_SERVICE_CONTRACT_RESOLVER_INTERFACE = "\\Magento\\Framework\\GraphQl\\Query\\Resolver\\BatchServiceContractResolverInterface";
```
Comment (VitaliyBoyko) [inline src/com/magento/idea/magento2plugin/magento/files/GraphQlResolver.java:10]: I did it in my PR. Please fix the only issue with a line in Quick Fix. Thank you!

Comment (konarshankar07) [reply src/com/magento/idea/magento2plugin/magento/files/GraphQlResolver.java:10]: Yes, But I have improved the quick fix with adding a select box to select one of the expected interfaces currenly quick fix is only added for `\Magento\Framework\GraphQl\Query\ResolverInterface`. 
Thanks

Comment (VitaliyBoyko) [reply src/com/magento/idea/magento2plugin/magento/files/GraphQlResolver.java:10]: Cool, great job

Comment (VitaliyBoyko) [reply src/com/magento/idea/magento2plugin/magento/files/GraphQlResolver.java:10]: Is this still in progress?

**Comment by konarshankar07** [comment]
I've tested regading the new line issue and I'm not able to reproduce in the debugger
https://www.loom.com/share/231427f592744559b6bb79ce1faa936b
Please check
Do you know how we can debug the plugin in PhpStorm ?
Thanks

**Comment by VitaliyBoyko** [comment]
Unfortunately, we have not toolset for PHP storm.
Let me check it once again.

**Comment by VitaliyBoyko** [comment]
Being reproduced in IntelliJ as well:
Apr-20-2020 15-56-58

**Comment by VitaliyBoyko** [comment]
@konarshankar07 
I'll try to fix it.

**Comment by konarshankar07** [comment]
Sorry, I forgot to mention you that now I'm able to reproduce this issue and working on it

**Pull request merged**